GitTank/ViewModels/TabWithLogsViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace GitTank.ViewModels
+{
+    public class TabWithLogsViewModel : BaseViewModel
+    {
+        public string Header { get; set; }
+
+        private string _outputInfo;
+        public string OutputInfo
+        {
+            get => _outputInfo;
+            set
+            {
+                if (value is not null)
```

Comment:
I would compare it to equals because null might be used to cleanup the logs output.